functions/powersOfTwo/powersOfTwo.js

```diff
@@ -1,9 +1,14 @@
-export const powersOfTwo = (n) => {
-  const arr = []
-  
-  for (let i=0; i<=n; i++) {   
-    arr.push(Math.pow(2, i))
+/** 
+ * @param {number} indicator
+ * @returns {array}
+ */
+
+export const powersOfTwo = (indicator) => {
+  const arr = [];
+
+  for (let index = 0; index <= indicator; index++) {
```

Comment:
The name "count" is more appropriate here because you are incrementing the counter
The name "index" suits when you use it in an iteration of an array